sirepo/template/raydata.py

```diff
@@ -173,19 +179,17 @@ def _metadata(data):
     return res
 
 
-def _scan_info(scan_uuid, metadata=None):
+def _scan_info(scan_uuid, selected_columns, metadata=None):
     m = metadata
     if not m:
-        m =  catalog()[scan_uuid].metadata
-    return PKDict(
-        uid=scan_uuid,
-        suid=_suid(scan_uuid),
-        owner=m['start']['owner'],
-        start=m['start']['time'],
-        stop=m['stop']['time'],
-        T_sample_=m['start'].get('T_sample_'),
-        sequence_id=m['start']['sequence id'],
-    )
+        m = catalog()[scan_uuid].metadata
+    d = PKDict()
+    d.start = m['start']['time']
```

Comment:
Use const datastructure to keep track default columns.

Reply:
e64969d8a3a9303ebcf97566971db0411515341f